### PR TITLE
Tests in yaml

### PR DESCRIPTION
**Pull request by MalinAhlberg**
### Expressing compliance tests in yaml
  
  This is a suggestion of how to translate the tests from python code into yaml.
  Test data is given in separate csv files. It is also possible (but not required) to give references to vcf files.
  
  - An example of a [test file](https://github.com/NBISweden/beacon-api-tests/blob/feature/test-config/tests/test-v101-counts.yaml).
  - An example of [test data](https://github.com/NBISweden/beacon-api-tests/blob/feature/test-config/tests/testdata.csv).
  
  
  There is also a [schema](https://github.com/NBISweden/beacon-api-tests/blob/feature/test-config/tests/schema.yaml) (also in yaml), describing what the tests should look like.
  Using this, one can easily verify that a newly added tests is correct (syntax-wise), by running
  `python beacon_api_tester.py --validate_tests tests/test1.yaml`
  
  
  ### Changes made
  Test data
  - translate the tests from python code to yaml (as proof of concept)
  - schema (`tests/schema.yaml`)
  - optional vcf ref
  - data for tests in separate files (`tests/testdata.csv`)
  
  
  Code
  - code for running yaml tests
  - code for testing that yaml tests complies with schema
  
  
  Addresses #32.
  
  Although there is still more job to do regarding the code updates for this, it would be nice to start
  a discussion about the format.

### CODE REVIEW

**Comment by mbaudis** [comment]
@MalinAhlberg This looks very nice! So the tests in YAML, and the data in `testdata.csv`, instead of the VCF?

**Comment by MalinAhlberg** [comment]
@mbaudis Yes, exactly. There is still a possibility to add a reference to a vcf line, in addition to the `csv` data (although we are still not sure about the details for that).

File: utils/generate_mdschema.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+"""Generate markdown information about tests from the test schema."""
```
Comment (viklund) [inline utils/generate_mdschema.py]: I think this script should put something like the following either at the start or the end of the resulting file, as a guide to the reader that one shouldn't change the markdown file.

```Generetade with utils/generate_mdschema.py from schema.yaml on DATE```

Maybe even include a note about not editing the file directly but rather just update the underlying schema and run the script again.

Comment (MalinAhlberg) [reply utils/generate_mdschema.py]: Added in f20a63b.

**Pull request merged**